spec/services/find_programme_activities_spec.rb

```diff
@@ -8,7 +8,20 @@
   let!(:extending_organisation_programme) { create(:programme_activity, extending_organisation: other_organisation) }
   let!(:other_programme) { create(:programme_activity) }
 
+  before(:each) do
+    Bullet.add_whitelist type: :unused_eager_loading, class_name: "Activity", association: :parent
```

Comment:
I've said before I am not super clear on bullet but I feel like this is just being set globally, how do we check it is removed after `each`? When I look at `Bullet.whitelist` 'after(:each)` and `after(:all)` I still see `:parent` which I think is this config?

Reply:
Sorry, I should have made myself more clear as I had left some new thoughts on this in the commit message:

> I tested that setting the whitelist in this way does not overwrite the global Bullet configuration. I ran 2 tests, the former with the whitelist change, and the latter without. The first file passes and the second fails. This confirms that the whitelist did not persist to other subsequant tests.

I'm surprised that the `Bullet.whitelist` is different in other tests.

Reply:
I think what we do here is tied to where whitelist config here which is the subject of the other comment thread here https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/494#pullrequestreview-441483206